base/docs/Docs.jl

```diff
@@ -304,15 +304,15 @@ function docm(meta, def, define = true)
 
     fexpr(def′)                && return funcdoc(meta, def, def′, namify(def′))
     isexpr(def′, :call)        && return funcdoc(meta, nothing, def′, namify(def′))
-    isexpr(def′, :type)        && return typedoc(meta, def, namify(def′.args[2]))
+    isexpr(def′, :type)        && return typedoc(meta, def, def′, namify(def′.args[2]))
     isexpr(def′, :macro)       && return namedoc(meta, def, symbol("@", namify(def′)))
     isexpr(def′, :abstract)    && return namedoc(meta, def, namify(def′))
     isexpr(def′, :bitstype)    && return namedoc(meta, def, def′.args[2])
     isexpr(def′, :module)      && return namedoc(meta, def, def′.args[2])
     isexpr(def′, :(=), :const) && return namedoc(meta, def, namify(def′))
 
-    isexpr(def′, :macrocall) && (def = namify(def′))
-    return objdoc(meta, def)
+    isexpr(def′, :macrocall) && (def′ = namify(def′))
```

Comment:
Yeah, I've removed that line in the bindings PR #12546. I can remove it in this one instead though if you'd rather.